### DISCUSSION TITLE
Bad user

### DISCUSSION THREAD

**Discussion by NodXygen** [revoxhere/duino-coin#1618] [General]
Today i had build my mining rig a master and slaves in i2c. But i try to mining and a cant.

Bad user is the response from the API.
I need help.

User: NodXygen

**Comment by petersimeth** [comment]
Should work, according to the API
https://server.duinocoin.com/users/NodXygen

{"balance":725.5524526432162,"created":"03/08/2022 10:03:44","last_login":1676679532,"stake_amount":1642.603,"stake_date":1677196800,"username":"NodXygen","verified":"yes","verified_by":"Technik","verified_date":1659624223,"warnings":0}

**Reply by NodXygen** [reply]
Resolved

**Comment by NodXygen** [comment]
Hello Revoxhere,
I getting this " Feb 18 04:09:23 ARRAKIS python3[366]: 04:09:23  net0
MOTD: BAD,User banned"

Thanks for replying.


Peter Simeth ***@***.***> escreveu no dia sábado, 18/02/2023
à(s) 03:19:

**Reply by NodXygen** [reply]
Resolved

**Comment by NodXygen** [comment]
Hello, i cant mine, can you help me, please? BAD, User banned.
I just using normally my rig.

Thanks in advance:
Helder Simões

[image: image.png]

Peter Simeth ***@***.***> escreveu no dia sábado, 18/02/2023
à(s) 03:19:
